devops/lib/utils.py

```diff
@@ -36,11 +36,13 @@ def load_env_settings(env: str) -> Settings:
 
 
 def list_envs() -> List[str]:
-    envs = []
-
-    for path in Path("envs").iterdir():  # type: Path
-        if path.is_dir() and not path.name.startswith("__"):
-            envs.append(path.name)
+    envs = [
```

Comment:
Well if you're going for shorthand syntax .. `return [`

Reply:
Just to make things a bit easier to debug. But if necessary, it can be changed.